requirements-test-libraries.txt

```
@@ -19,6 +19,7 @@ dash==2.3.0
 dash-bootstrap-components==1.0.3
 dash-uploader==0.6.0
 fabric==2.7.0
+fiona==1.8.21

```

Since fiona [doesn't readily support Windows](https://fiona.readthedocs.io/en/latest/README.html#windows). 
```suggestion
fiona==1.8.21; sys_platform != 'win32'
```